main.py

```diff
@@ -188,10 +190,12 @@ def __init__(self, x=0, y=0):
             "l": ((8, 8), (16, 8), (24, 8)),
         }
         self.alive = True
+        self.active = False
 
     def update(self):
         if not self.alive:
             return
+        if not self.active: return
```

Comment:
_Maybe_ this could fix the failure?

```suggestion
        if not self.active:
            return
```

Reply:
I like this kind of one liner personally `if <condition>: statement`, if it is only one statement, like return or print, but I guess flake8 doesn't

Reply:
Indeed. I'm also a bit more +1 to the one-liner, but we gotta respect flake8' style :man_shrugging: